lambdas/es/indexer/index.py

```diff
@@ -297,6 +297,63 @@ def do_index(
         logger_.debug("%s indexed as package (%s)", key, event_type)
 
 
+def _try_parse_date(s: str) -> datetime.datetime:
+    # XXX: do we need to support more formats?
+    # XXX: do we need to preserve UTC timezone?
```

Comment:
why not?

Reply:
1. `fromisoformat()` can't parse ISO format string that ends on `Z`
2. if ES treats timestamps without timezone the same way as with UTC then probably there is no point to complicate things preserving timezone